src/njoy21/input/MIXR/Card5/Za.hpp

```diff
@@ -0,0 +1,10 @@
+struct Za {
+  using Value_t = int;
+  static std::string name() { return "za"; }
+  static std::string description(){
+    return
+      "The za argument is calculated to be 1000 * Z + A, where Z is the\n"
+      "charge nuber and A is the atomic mass number for the mixture.";
+  }
+  static bool verify( const Value_t& z ){ return z > 0; }
```

Comment:
A ZA of 1,000,000,000 is not valid. In other places, I've maxed out the ZA to be 200000.